javiereguiluz/easyadmin-bundle/1.17/config/packages/easy_admin.yaml

```diff
@@ -1,6 +0,0 @@
-#easy_admin:
-#    entities:
-#        # List the entity class name you want to manage
-#        - App\Entity\Product
-#        - App\Entity\Category
-#        - App\Entity\User
```

Comment:
Example files are mostly the hardest to be converted 🤔 Not sure if even possible.